Packs/AzureSentinel/Integrations/AzureSentinel/AzureSentinel.py

```diff
@@ -1361,11 +1697,54 @@ def replace_tags_threat_indicator_command(client, args):
     )
 
 
+def list_alert_rule_command(client: AzureSentinelClient, args: Dict[str, Any]) -> CommandResults:
+    limit = int(args.get('limit', 50))
```

Comment:
why not to use `arg_to_number`?

Reply:
Because in line 1716 there will be an error that len ​​cannot be compared to something that can be None